src/Features/VisualBasic/Portable/Structure/Providers/MultiLineIfBlockStructureProvider.vb

```diff
@@ -14,9 +14,59 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Structure
                                                   spans As ArrayBuilder(Of BlockSpan),
                                                   options As OptionSet,
                                                   cancellationToken As CancellationToken)
-            spans.AddIfNotNull(CreateBlockSpanFromBlock(
-                               node, node.IfStatement, autoCollapse:=False,
-                               type:=BlockTypes.Conditional, isCollapsible:=True))
+
+            ' The full block: eg If ... Then ... End If
+            spans.AddIfNotNull(CreateBlockSpanFromBlock(node, node.IfStatement,
+                                                        autoCollapse:=False,
+                                                        type:=BlockTypes.Conditional,
+                                                        isCollapsible:=True))
+
+            ' Attempt to make the code block after the If...Then collapsable.
+            Make_IfThenCodeBlockCollapsible(node, spans)
+
+            ' Make the ElseIfs collapsable.
+            Make_ElseIfCodeBlocksCollapsible(node, spans)
+
+            ' Make the Else section collapsable.
+            Make_ElseCodeBlockCollapsible(node, spans)
+
+        End Sub
+
+        Private Shared Sub Make_ElseCodeBlockCollapsible(node As MultiLineIfBlockSyntax, spans As ArrayBuilder(Of BlockSpan))
+            If node.ElseBlock IsNot Nothing Then
+                spans.AddIfNotNull(CreateBlockSpanFromBlock(node.ElseBlock, "Else ... ",
```

Comment:
we generally do not hardcode this in.  we have things like ```SpaceEllipses```.  But, that said, it's unclear to me why we're hardcoding anything in here.  Can you show screenshots showing what the new experience looks like?